cookbooks/fail2ban/recipes/service.rb

```diff
@@ -0,0 +1,16 @@
+#
+# Cookbook Name:: fail2ban
+# Recipe:: services
+#
+#
+
+ey_cloud_report "Fail2Ban-service" do
+  message "Fail2Ban service & monitoring"
+end
+
+# enabling the service
+service 'fail2ban' do
+  supports [:status => true, :restart => true]
+  action [:enable]
+  status_command "/etc/init.d/fail2ban status | grep -q 'status: started'"
```

Comment:
The grep command is wrong because /etc/init.d/fail2ban on Ubuntu 18.04 uses systemctl. The chef recipes work with or without this line. I suggest we just remove status_command.